Cesium3DTiles/src/QuantizedMeshContent.cpp

```diff
@@ -651,6 +677,92 @@ static std::vector<std::byte> generateNormals(
   return normalsBuffer;
 }
 
+// Create texture coordinates for tiles such that the texture fits exactly
+static void createFittedTextureCoordinatesForTile(
+    CesiumGltf::Model& model,
+    uint32_t positionsAccessorId,
+    uint32_t textureCoordinatesId,
+    const CesiumGeospatial::GlobeRectangle& rectangle) {
+  // TODO: How often exactly do these types of checks need to be made?
+  // If the accessor id is valid, then are the buffer view and buffer ids
+  // implicitly valid?
```

Comment:
No, a valid accessor doesn't guarantee the objects it references are valid. Easiest thing is to create an `AccessorView` and then check the `status`.

In fact, the code below won't work if the glTF has positions interleaved with other attributes. `AccessorView` takes care of all that.

Reply:
There's also an `AccessorWriter`.